src/constants/gameplay.asm

```diff
@@ -165,6 +165,9 @@ SWORD_ANIMATION_STATE_SWING_MIDDLE equ $03
 SWORD_ANIMATION_STATE_SWING_END    equ $04
 SWORD_ANIMATION_STATE_HOLDING      equ $05
 
+; Charge amount needed for spin attack
+MAX_SWORD_CHARGE equ $28
```

Comment:
This name could imply that this is the charge level for the sword throwing beams (when on full hearts). Maybe `SWORD_MAX_SPIN_CHARGE` would be a better name?

Reply:
I do not understand. When is the sword throwing beams?
New name is god.

Reply:
The level 2 sword (the one found in the Search Seashells Mansion) throws beams when all hearts are full :)

Reply:
I looked over all usages of wSwordCharge.
I think the max charge for this beams is the same value. $28

Reply:
So rename to SWORD_MAX_SPIN_CHARGE?